README.md

```diff
@@ -916,45 +901,38 @@ Optional configurations :
         schema.profileSchema.fieldOrder.push('address');
         onSuccess(schema);
     }
-    onFailure callback accepts a errorObject that can be used to show form level or field level errors
     ```
- - **preSubmit :** Callback used primarily to modify the request parameters sent to Okta API.
+ - **preSubmit:** Callback used primarily to modify the request parameters sent to Okta API.
 
     ```javascript
-    /**
-     The callback function is passed 3 arguments:
-     @params : postData, onSuccess, onFailure
-     1) @param {Object} postData: form data that will be posted to the registration API.
-     2) @param {Function} onSuccess: success callback.
-     3) @param {Function} onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
-    **/
+     // The callback function is passed 3 arguments: postData, onSuccess, onFailure
+     // 1) postData: form data that will be posted to the registration API.
+     // 2) onSuccess: success callback.
+     // 3) onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
     preSubmit: function (postData, onSuccess, onFailure) {
       // This example will add @companyname.com to the email if user fails to add it during registration
       return postData.username.indexOf('@acme.com') > -1 ? username : username + '@acme.com';
```

Comment:
Usually in sample code, it is best to split out logic into multiple lines for the reader:

```javascript
preSubmit: function (postData, onSuccess, onFailure) {
  if (postData.username.indexOf('@acme.com') > 1) {
     return postData.username;
  } else {
    return postData.username + '@acme.com';
  }
}
```

Reply:
I can update that, but I got this example from the transform Username function https://github.com/okta/okta-signin-widget#username-and-password